client/src/partials/journal/JournalPosting.service.js

```diff
@@ -0,0 +1,23 @@
+angular.module('bhima.services')
+  .service('JournalPostingService', JournalPostingService);
```

Comment:
Hum ... it is a service for user to post data from journal to general ledger so I think ```JournalPostingService``` is more expressive.